ui/app/src/services/audit.ts

```diff
@@ -59,8 +59,9 @@ export function fetchAuditLog(options: AuditOptions): Observable<PagedArray<Audi
   );
 }
 
-export function fetchAuditLogEntry(id: number): Observable<AuditLogEntry> {
-  return get<Api2ResponseFormat<{ auditLog: AuditLogEntry }>>(`/studio/api/2/audit/${id}`).pipe(
+export function fetchAuditLogEntry(id: number, siteId?: string): Observable<AuditLogEntry> {
+  const qs = toQueryString({ siteId });
+  return get<Api2ResponseFormat<{ auditLog: AuditLogEntry }>>(`/studio/api/2/audit/${id}${qs}`).pipe(
```

Comment:
Did you test this?? Without the `skipNull` siteId argument will be sent empty. Global shouldn't have a siteId
```suggestion
  const qs = toQueryString({ siteId }, { skipNull: true });
  return get<Api2ResponseFormat<{ auditLog: AuditLogEntry }>>(`/studio/api/2/audit/${id}${qs}`).pipe(
```

Reply:
I did, when null the queryString ends up like `?`, no siteId parameter